lib/sqlalchemy/sql/selectable.py

```diff
@@ -2067,7 +2092,7 @@ def alias(self, name: Optional[str] = None, flat: bool = False) -> CTE:
             :func:`_expression.alias`
 
         """
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325